src/Resources/views/tailwind_2/_table.html.twig

```diff
@@ -159,11 +159,13 @@
             </tr>
         {% endif %}
         {% for row in table.rows %}
+            {% set subTables = table.getSubTables(row) %}
             <tr class="whatwedo_table-row hover:bg-neutral-100 transition duration-500 color align-top"
-                {{ stimulus_action('araise/table-bundle/accordion', 'toggle', 'click') }}
+                {% if subTables %}
+                    {{ stimulus_action('araise/table-bundle/accordion', 'toggle', 'click') }}
+                {% endif %}
```

Comment:
@marcwieland95 Just saw that by accident.

Coming from a place of having evaluated and dealt with multiple Twig formatters for Twig project standards, I thought it might be useful to know, that having if-else clauses inside HTML Opening/Closing Tags can potentially lead to problems because formatters don't know what to do with it.

Following for example doesn't lead to the same problems:

```twig
{% set my_attribute = condition ? stimulus_action(...) : '' %}
<element {{ my_attribute }}></element>
```

Reply:
Thanks for your input @renestalder. I will update the code accordingly 👍

Reply:
@renestalder Alright. The changes are made. Would you like to take another look at it? 🙂